docs/releasenotes/release-1.127.md

```diff
@@ -1,35 +1,53 @@
-# v1.126.0
+* Special shout-outs to @acpana, @anhdle-sso, @cheftako, @ericpang777, @gemmahou, @haiyanmeng, @jasonvigil, @jingyih, @justinsb, @maqiuyujoyce, @nb-goog, @xiaoweim, @yuwenma, @ziyue-101 for their contributions to this release.
 
-** This version is not yet released; this document is gathering release notes for the future release **
+## Announcement 
 
-* ...
 
-* Special shout-outs to ... for their contributions to this release.
-TODO: list contributors with `git log v1.126.0... | grep Merge | grep from | awk '{print $6}' | cut -d '/' -f 1 | sort | uniq`
+## New Beta Resources (Direct Reconciler):
 
-## Announcement 
+* [`BigQueryAnalyticsHubListing`](https://cloud.google.com/config-connector/docs/reference/resource-docs/bigqueryanalyticshub/bigqueryanalyticshublisting)
 
-## New Beta Resources (Direct Reconciler):
+  * Manage the reference to a shared dataset that a publisher lists in a data exchange.
+
+* [`FirestoreDatabase`](https://cloud.google.com/config-connector/docs/reference/resource-docs/firestore/firestoredatabase)
+
+  * Manage the Firestore database (NoSQL). 
+
+* [`WorkstationConfig`](https://cloud.google.com/config-connector/docs/reference/resource-docs/workstations/workstationconfig)
 
-* `WorkstationConfig`
-* `Workstation`
-* `FirestoreDatabase`
+  * Manage the template that defines the settings for a workstation, such as the machine type, disk size, and tools.
+
+* [`Workstation`](https://cloud.google.com/config-connector/docs/reference/resource-docs/workstations/workstation)
+
+  * Manage the preconfigured development environment that administrators or platform teams define in a workstation configuration.
 
 ## New Fields:
 
-* `GkeHubFeatureMembership`
-  * Added `spec.configmanagement.management` field to enable Config Sync Auto Upgrade.
+* `BigQueryDataTransferConfig`
 
-* `Placeholder`
+  * Added `spec.scheduleOptionsV2` to customize the different types of data transfer schedule.
+  * Added `status.observedState.error` with detailed information about reason of the latest config failure.
 
 ## Modified Beta Reconciliation
 
 We migrated the following reconciliation from the TF-based or DCL-based controller to the new Direct controller to enhance the reliability and performance. The resource CRD is unchanged.
 
-* `GkeHubFeatureMembership` is now a direct resource.
 
-* `Placeholder`
+* `GkeHubFeatureMembership`
+
+  * Added `spec.configmanagement.management` to enable Config Sync Auto Upgrade.
```

Comment:
Personally I would add to "New Fields" and to "modified reconciler".  They are telling the user different things: the new field is saying "there's new functionality available".  The "Modified Beta Reconciliation" tells the user that if they see a weird regression, it might be because of this.

Reply:
I think the field should be changed. I would keep the reconciliation warning here. Might be worth being more explicit. Eg Fixed bug where false value was being ignored.

Reply:
Thanks! Added the item in both places.